log/keyvalue.go

```diff
@@ -351,16 +351,19 @@ func Bool(key string, value bool) KeyValue {
 }
 
 // Bytes returns a KeyValue for a []byte value.
+// The passed slice must not be changed after it is passed.
 func Bytes(key string, value []byte) KeyValue {
 	return KeyValue{key, BytesValue(value)}
 }
 
 // Slice returns a KeyValue for a []Value value.
+// The passed slice must not be changed after it is passed.
```

Comment:
Nit, but there is technically no slice passed (`value` is a `...attribute`). How about:

```suggestion
// The value must not be changed after it is passed.
```

Reply:
You can pass a slice using `v...` syntax.
If you just pass `v1,v2,v3` then the compiler creates the slice.
I find the comment to be precise. 
For reference a similar comment from [`slog.GroupValue`](https://pkg.go.dev/log/slog#GroupValue) :

> The caller must not subsequently mutate the argument slice.

Side note: that the same comment is already for `SliceValue` and `MapValue`.